### PR TITLE
Add test for docvalue_fields retrieval of murmur3

### PR DESCRIPTION
**Pull request by lkts**
This PR adds a test for obtaining the actual hash value from `murmur3` field.

### CODE REVIEW

File: plugins/mapper-murmur3/src/yamlRestTest/resources/rest-api-spec/test/mapper_murmur3/10_basic.yml [hunk 1]
```diff
@@ -104,3 +104,26 @@ setup:
           fields: [ foo.hash ]
 
   - match: { hits.hits.0.fields: {foo.hash: ["foo"]} }
+
+---
+"Murmur3 docvalue_fields api":
```
Comment (martijnvg) [inline plugins/mapper-murmur3/src/yamlRestTest/resources/rest-api-spec/test/mapper_murmur3/10_basic.yml:109]: Maybe we also need a test with synthetic source enabled?  The `test` index here doesn't synthetic source enabled. Just to double check murmur field works as expected.

Comment (lkts) [reply plugins/mapper-murmur3/src/yamlRestTest/resources/rest-api-spec/test/mapper_murmur3/10_basic.yml:109]: My thinking was that it does not matter but the test would serve as a documentation for that, i'll add it.

**Comment by lkts** [comment]
buildkite test this

**Comment by lkts** [comment]
@elasticmachine update branch

**Review by martijnvg — APPROVED** [review]
lgtm

**Pull request merged**